Basic/APIDiorama/Assets/UseCaseSamples/SelectionScreen/Scripts/SceneSelectionElement.cs

```diff
@@ -0,0 +1,56 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+namespace Unity.Netcode.Samples.APIDiorama
+{
+    /// <summary>
+    /// Allows to select a scene in the SelectionScene
+    /// </summary>
+    internal class SceneSelectionElement : MonoBehaviour, IPointerEnterHandler, IPointerExitHandler
+    {
+        [SerializeField] Button m_SceneButton;
+        [SerializeField] TMP_Text m_TitleLabel;
+        [SerializeField] Image m_ContrastImage;
+
+        internal void Setup(SelectableScene selectableScene)
+        {
+            m_SceneButton.onClick.RemoveAllListeners();
+            m_SceneButton.onClick.AddListener(() => OnClick(selectableScene.SceneName));
```

Comment:
I'm getting an error when I try to open a scene from the SelectionScreen Scene. I've tried deleting my library folder and reopening the project but I still get the error 🤔 

```
Cannot load scene: Invalid scene name (empty string) and invalid build index -1
UnityEngine.SceneManagement.SceneManager:LoadScene (string,UnityEngine.SceneManagement.LoadSceneMode)
Unity.Netcode.Samples.APIDiorama.SceneSelectionElement:LoadScene (string) (at Assets/UseCaseSamples/SelectionScreen/Scripts/SceneSelectionElement.cs:53)
Unity.Netcode.Samples.APIDiorama.SceneSelectionElement:OnClick (string) (at Assets/UseCaseSamples/SelectionScreen/Scripts/SceneSelectionElement.cs:48)
Unity.Netcode.Samples.APIDiorama.SceneSelectionElement/<>c__DisplayClass3_0:<Setup>b__0 () (at Assets/UseCaseSamples/SelectionScreen/Scripts/SceneSelectionElement.cs:21)
UnityEngine.EventSystems.EventSystem:Update ()
```

Reply:
@jilfranco-unity  Looks like I didn't commit the published scene after the last change, thanks for catching it up!
It should be fixed now

Reply:
yes it works now, thank you! :)